addons/udes_stock/models/stock_move.py

```diff
@@ -687,3 +687,14 @@ def _refactor_action_by_maximum_quantity(self, maximum_qty):
                 new_pickings += picking._backorder_movelines(mls, dest_picking)
 
         return self | new_pickings.mapped("move_lines")
+    
+    def _split(self, qty, restrict_partner_id=False):
+        """
+        If the moves have movelines we do not want to refactor the moves when we split them, as
+        the movelines will not be refactored.
+        """
+        if self.mapped("move_line_ids"):
```

Comment:
It looks like self is always 1 record as even in core odoo is treated like that self.product_uom_qty etc.
I think don't need the mapped in this case,

Reply:
changed :slightly_smiling_face: